client/starwhale/core/dataset/type.py

```diff
@@ -232,50 +236,32 @@ def type(self) -> ArtifactType:
     def mime_type(self) -> MIMEType:
         return MIMEType(self._mime_type)
 
-    @classmethod
-    def reflect(cls, raw_data: bytes, data_type: t.Dict[str, t.Any]) -> BaseArtifact:
-        if not isinstance(raw_data, bytes):
-            raise NoSupportError(f"raw data type({type(raw_data)}) is not bytes")
-
-        # TODO: support data_type reflect
-        dtype = data_type.get("type")
-        mime_type = MIMEType(data_type.get("mime_type", MIMEType.UNDEFINED))
-        shape = data_type.get("shape", [])
-        encoding = data_type.get("encoding", "")
-        display_name = data_type.get("display_name", "")
-
-        if dtype == ArtifactType.Text.value:
-            _encoding = encoding or Text.DEFAULT_ENCODING
-            return Text(content=raw_data.decode(_encoding), encoding=_encoding)
-        elif dtype == ArtifactType.Image.value:
-            return Image(
-                raw_data, mime_type=mime_type, shape=shape, display_name=display_name
-            )
-        elif dtype == ArtifactType.Audio.value:
-            return Audio(
-                raw_data, mime_type=mime_type, shape=shape, display_name=display_name
-            )
-        elif dtype == ArtifactType.Video.value:
-            return Video(
-                raw_data, mime_type=mime_type, shape=shape, display_name=display_name
-            )
-        elif not dtype or dtype == ArtifactType.Binary.value:
-            return Binary(raw_data)
-        elif dtype == ArtifactType.Link.value:
-            return cls.reflect(raw_data, data_type["data_type"])
-        else:
-            raise NoSupportError(f"Artifact reflect error: {data_type}")
-
     def to_bytes(self, encoding: str = "utf-8") -> bytes:
+        return self.fetch_data(encoding)
+
+    def clear_bytes(self) -> None:
+        self._bytes = bytes()
+        if isinstance(self.fp, (bytes, io.IOBase)):
+            self.fp = ""
+
+    def fetch_data(self, encoding: str = "utf-8") -> bytes:
```

Comment:
`fetch_data` -> `to_bytes` ?

Reply:
> `fetch_data` -> `to_bytes` ?
We have  `to_bytes` method at line 239. `fetch_data` is more explicit than `to_bytes` when you know it is time consuming. 
`fetch_data` is designed for developers, while `to_bytes` is deisgned for users.